clusters/app.ci/vault/manifests.yaml

```diff
@@ -56,7 +73,7 @@ metadata:
   name: vault
   namespace: vault
 spec:
-  replicas: 3
+  replicas: 0
```

Comment:
I'll remove the deployment manifest in a follow-up, this is to make sure that creation of statefulset and downscaling of deployment happen together